webrender/res/ps_text_run.glsl

```diff
@@ -22,12 +22,18 @@ VertexInfo write_text_vertex(vec2 clamped_local_pos,
                              PictureTask task,
                              RectWithSize snap_rect,
                              vec2 snap_bias) {
-#if defined(WR_FEATURE_GLYPH_TRANSFORM) || !defined(WR_FEATURE_TRANSFORM)
     // Ensure the transform does not contain a subpixel translation to ensure
     // that glyph snapping is stable for equivalent glyph subpixel positions.
-    scroll_node.transform[3].xy = floor(scroll_node.transform[3].xy + 0.5);
+#if defined(WR_FEATURE_GLYPH_TRANSFORM)
+    bool remove_subpx_offset = true;
+#else
+    bool remove_subpx_offset = scroll_node.is_axis_aligned;
```

Comment:
shouldn't this be the opposite of `is_axis_aligned`?

Reply:
The previous logic would include this conditional compilation on ```!defined(WR_FEATURE_TRANSFORM)``` which is the same as ```is_axis_aligned```, I think.